Makefile

```diff
@@ -0,0 +1,40 @@
+NODE_BIN=./node_modules/.bin
+
+help:
+	@echo '                                                                                     		'
+	@echo 'Makefile for the edX Cloudmanager SDK                                                            ' 
+	@echo '                                                                                     		'
+	@echo 'Usage:                                                                               		'
+	@echo '    make requirements                 install requirements for local development     		'
+	@echo '    make clean                        delete generated byte code and coverage reports		'
+	@echo '    make validate_python              run Python unit tests and quality checks       		'
```

Comment:
Neither of this or `fast_validate_python` run unit tests right now.  We should update the docs.